core/src/main/java/org/apache/carbondata/core/scan/model/QueryModel.java

```diff
@@ -118,6 +118,8 @@
    */
   private boolean isDirectVectorFill;
 
+  private boolean readOnlyDelta;
```

Comment:
please add comment to describe this variable

Reply:
ok